tests/SampleApps/DotNetCore/NetCoreApp21.WebApp/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace NetCoreApp21.WebApp
```

Comment:
An app with `.` in its name..this will generate a startup file with `.` in it...so its `NetCoreApp21.WebApp.dll`